src/components/search/no-results-message.vue

```diff
@@ -2,8 +2,8 @@
   <span
     v-if="$x.noResults"
     v-html="$t('noResults.message', { query: $x.query.search })"
-    class="x-no-results-message x-message x-text"
-    :class="{ 'x-font-size--05': $x.device === 'desktop' }"
+    class="x-text1 x-no-results-message x-message"
+    :class="{ 'x-text1-lg': $x.device === 'desktop' }"
```

Comment:
question: Wasn't using tailwind responsive prefix one of the reasons to create the XDS using tailwind? Why do we still use dynamic classes?
```suggestion
    class="desktop:x-text1-lg"
```

Reply:
True, I'll search for more and change them.